num_obstacles: 5
x_min: -5.0
x_max: 5.0
y_min: -2.5
y_max: 10.0
obstacles:
  0:
    x: 1.0
    y: 0.0
    r: 0.25
  1:
    x: 2.0
    y: 0.5
    r: 0.25
  2:
    x: 1.0
    y: 0.5
    r: 0.25
  3:
    x: 2.5
    y: 1.0
    r: 0.25
  4:
    x: 3.0
    y: -1.0
    r: 0.25